src/aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -313,7 +314,8 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
         if 'JOB DONE' in line:
             break
     else:
-        logs.error.append('ERROR_OUTPUT_STDOUT_INCOMPLETE')
+        if crash is None:
+            logs.error.append('ERROR_OUTPUT_STDOUT_INCOMPLETE')
```

Comment:
Could merge the separate else and if
```suggestion
    elif crash is None:
        logs.error.append('ERROR_OUTPUT_STDOUT_INCOMPLETE')
```

Reply:
I actually get an identation error when using `elif` instead of `else`. Moving it inside the for loop does not give the same results.

Reply:
Well, you also have to dedent the `logs.error.append` statement properly. It is a 100% fine to do 
```python
if ..:
    something
elif ...:
    something
```

Reply:
I agree, but there is a `break` statement before, so placing `elif` inside the main loop (indenting it) produces a different result (i.e. tests are not passing).

Reply:
Ok, got the fix for it w/o `elif`

Reply:
Ah, I see now. The `else` was not the secondary clause to an `if` but a `for` loop. Indeed, then `elif` is not valid syntax. Fine to leave as is then